Library/Formula/memo.rb

```diff
@@ -0,0 +1,20 @@
+class Memo < Formula
+  url "http://www.getmemo.org/memo-1.4.tar.gz"
+  homepage "http://www.getmemo.org"
+  sha1 "0b4c1b22fac0644b54d7de1e89cb20a54c8fff25"
+
+
+  def install
+    system "make", "PREFIX=#{prefix}"
+    bin.install "memo"
+    man1.install "memo.1"
+    prefix.install
+  end
+
+
+  test do
+    system "#{bin}/memo", "-a", "Lorem ipsum dolor sit amet."
+    system "#{bin}/memo", "-m", "1"
+    system "#{bin}/memo", "-d", "1"
```

Comment:
Let's avoid touching the user's default memo database:

> It's also possible to set memo path using an environment variable. For example: export MEMO_PATH=/path/to/some/file. If MEMO_PATH environment variable is set that will be used. If it's not set, Memo will try to read $HOME/.memorc. If the file does not exist or has errors, Memo will use $HOME/.memo file to store the notes.

You can set `ENV["MEMOPATH"] = testpath`.